util/cpb/main.go

```diff
@@ -84,14 +84,32 @@ func getMetadata() (m *metadata, err error) {
 		manifestDir:     "/manifests",
 	}
 
+	// Exclude device filesystems and pseudo filesystems from filesystems looking for metadata
+	excludeDir := []string{
+		"/dev",
+		"/proc",
+		"/sys",
+	}
+
 	// Traverse the filesystem looking for metadata
 	err = filepath.Walk(".", func(path string, info os.FileInfo, err error) error {
 		if err != nil {
 			fmt.Printf("prevent panic by handling failure accessing a path %q: %v\n", path, err)
 			return nil
 		}
 		if info.IsDir() {
-			fmt.Printf("skipping a dir without errors: %+v \n", info.Name())
+			absPath, err := filepath.Abs(path)
+			if err != nil {
+				fmt.Printf("couldn't get the absolute path %q: %v\n", path, err)
+				return nil
+			}
+			for _, v := range excludeDir {
```

Comment:
nit: We could use a hash set for an O(1) lookup here, but since we have a relatively small and constant number of exclusions, I'm not too worried about being ultra efficient here.